### ISSUE TITLE
[Bug][iOS] Label Html  with vietnamese

### ISSUE THREAD

**Issue by gcomm-trile** [xamarin/Xamarin.Forms#8844]
I issued with Label Html  Text = <P><strong> ánh trăng</strong></p> in iOS( work fine in Android)
Xamarin.Form 4.3.0.991221
![Simulator Screen Shot - iPhone 8 Plus - 2019-12-11 at 13 46 54](https://user-images.githubusercontent.com/58388336/70598594-c3a19f80-1c1d-11ea-9b29-a2699bc2899e.png)

**Comment by Omarkth** [comment]
Same Problem with Arabic text. Encoding is missed up in iOS:
```
<StackLayout>
    <Label
        Text="نص عربي" />
    <BoxView
        HeightRequest="1"
        Color="Red" />
    <Label
        TextType="Html"
        Text="نص &lt;b&gt;عربي&lt;/b&gt;" />
    <BoxView
        HeightRequest="1"
        Color="Red" />
    <Label
        TextType="Html">
        <![CDATA[
        نص <b>عربي</b>
        ]]>
    </Label>
</StackLayout>
```
Here is how it looks:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/822615/70850292-55651300-1e91-11ea-81d4-cb0be03bbbe5.png">

It works just fine on Android.

**Comment by bares43** [comment]
Same problem here with Czech characters: ěščřžýáíé. On Android it works just fine.

**Comment by Sasha111ua** [comment]
same happens to Ukrainian and Russian

**Comment by bares43** [comment]
It seems it's fixed in new XF 4.5.0.356

**Comment by Sasha111ua** [comment]
> It seems it's fixed in new XF 4.5.0.356

Confirm for ukr and rus

**Comment by gcomm-trile** [comment]
confirm for vietnamese
